hack/verify-publishing-bot.py

```diff
@@ -48,6 +48,9 @@ def main():
     rootdir = os.path.dirname(__file__) + "/../"
     rootdir = os.path.abspath(rootdir)
 
+    # staging directories that do not need to be published yet
+    publishing_exceptions = ["cloud-provider-vsphere"]
```

Comment:
cc @nikhita 

Adding the vsphere staging directory as an exception for now until we sort out https://github.com/kubernetes/publishing-bot/issues/156

Reply:
/cc @dims @sttts

Reply:
What does it imply to be on this list? This must be documented.

Reply:
I don't think we'll need this anymore, proposed changes in https://github.com/kubernetes/enhancements/pull/952